public_html/index.html

```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
```

Comment:
I like that you added the locale. good catch l!

Reply:
*-*